tests/api/test_cluster.cpp

```diff
@@ -23,7 +23,7 @@
 #include "src/firmware/riscv/wormhole/eth_l1_address_map.h"
 #include "src/firmware/riscv/wormhole/host_mem_address_map.h"
 #include "src/firmware/riscv/wormhole/l1_address_map.h"
-#include "src/firmware/riscv/wormhole/noc/noc_parameters.h"
+#include "noc/noc_parameters.h"
```

Comment:
How does `umd` resolve this file when its the top-level?  Isn't this breaking their build?

And, uh, this file was only used by tests and not by the actual code?  🤔

Reply:
Right, only used for unit tests ... cause they need to simulate the behavior of tt-metal driving their APIs.

No it doesn't break the build, because of the new include path I introduced.

This test probably should not have been hardcoded for wormhole.

Reply:
Willing to hold my nose and get this in quickly so we can make forward progress.  Also knowing that the UMD interface is being looked at elsewhere, so this shouldn't sit around for toooo long 🤞